index.md

```diff
@@ -1,19 +1,81 @@
 # Welcome to Life Monitor
 
-Life Monitor is a testing and monitoring service for scientific worflows that
-is being developed as part of the [EOSC-Life
-project](https://www.eosc-life.eu/). Please refer to the [Life Monitor
-repository](https://github.com/crs4/life_monitor) and [its
-wiki](https://github.com/crs4/life_monitor/wiki) for additional information.
+Life Monitor is a **workflow testing and monitoring service** for scientific
+workflows.  
 
-## API Specs
+For scientific workflows, the “collapse” over time of the software and services
+on which they depend for correct operation is destructive to their reusability
+and to the reproducibility of work for which they were used; in this case,
+"collapse" can be a change in API that is not backwards compatible, a regression
+in a tool whose version was not pinned, a change in URL of an external resource,
+etc.
 
-Life Monitor exposes its functionalities through a RESTful API defined with
-[OpenAPI](https://swagger.io/specification). Below you'll find links to the
-current version of the API, which corresponds to the `master` branch in the
-GitHub repository.
+Life Monitor aims to facilitate the creation, sharing, periodic execution and
+monitoring of workflow tests, ensuring that software collapse is detected and
+communicated to the authors, in the hope it will be solved thus extending the
+useful life of the workflows.
 
-* Formatted with RapiDoc:  [lm-openapi-rapidoc.html](./lm-openapi-rapidoc.html)
-* Static page automatically created with openapi-generator: [lm-openapi-html](./lm-openapi-html/index.html)
 
-You can find the raw YAML API specifications in [the source code repository](https://github.com/crs4/life_monitor/tree/master/specs).  
+## Goals
+
+* Provide a central aggregation point for your workflow test statuses and outputs
+  from various testing bots (e.g., TravisCI, GitHub Actions, your own
+  Jenkins instance, etc.).
+* Integrated Jenkins-based workflow test execution service.
+* Facilitate periodic automated execution of tests for Galaxy, Nextflow and CWL
+  workflows.
+* Web interface, CLI client, REST API
+
+
+## How to use it
+
+At the moment, the Life Monitor can be used through its [REST
+API](lm_api_specs).  **Follow the link** for [documentation and
+specifications](lm_api_specs).
```

Comment:
Redundant link to the specs. I think the first sentence is enough

Reply:
Ok.  To me the formatting doesn't make links stand out much, but ok.